lua/nvim-lsp-installer/servers/clangd.lua

```diff
@@ -0,0 +1,30 @@
+local server = require('nvim-lsp-installer.server')
+
+local root_dir = server.get_server_root_path('c-family')
+
+local install_cmd = [=[
+rm -r clangd;
```

Comment:
You could omit this actually, when installing a new server the entire `root_dir` is [wiped](https://github.com/williamboman/nvim-lsp-installer/blob/main/lua/nvim-lsp-installer/server.lua#L133-L135), so every install cmd is executed in a completely pristine state

Reply:
removed with 24806b21d4e142790eb0b7b4f6ffc9cd5d631571 
Thank you!